### ISSUE TITLE
Problem adding reactions - gapseq adapt add

### ISSUE THREAD

**Issue by estebano11** [jotech/gapseq#88]
Hi, I am trying to add some reactions to my models but I have error with the configuration of the program since I can't add it neither with the example provide by you. 
I have tried in different versions and it was impossible.
The error with the version **1.1 75ebaea** is the following one:

```
(gapseq) user@server~/bin/gapseq-1.1$ ./gapseq adapt add 14DICHLORBENZDEG-PWY toy/myb71.RDS
Loading model files toy/myb71.RDS 
                     id     id.type                      db.rxn
1: 14DICHLORBENZDEG-PWY metacyc pwy              DIOXYXANFL-RXN
2: 14DICHLORBENZDEG-PWY metacyc pwy            CHLOROBENDDH-RXN
3: 14DICHLORBENZDEG-PWY metacyc pwy          CHLOROCATDIOXY-RXN
4: 14DICHLORBENZDEG-PWY metacyc pwy           DIENELACHYDRO-RXN
5: 14DICHLORBENZDEG-PWY metacyc pwy           CHLOROMALERED-RXN
6: 14DICHLORBENZDEG-PWY metacyc pwy MALEYLACETATE-REDUCTASE-RXN
7: 14DICHLORBENZDEG-PWY metacyc pwy                   RXN-18360
8: 14DICHLORBENZDEG-PWY metacyc pwy                   RXN-18361
9: 14DICHLORBENZDEG-PWY metacyc pwy                   RXN-18362
                                                                                                          seed
1:                                                                                                            
2:                                                                                                            
3:                                                                                                            
4: rxn02747 rxn03800 rxn03801 rxn04650 rxn04653 rxn07908 rxn07909 rxn15998 rxn16082 rxn16084 rxn31706 rxn31716
5:          rxn02141 rxn02142 rxn03656 rxn04663 rxn06904 rxn07595 rxn15999 rxn16000 rxn16085 rxn16086 rxn20159
6:          rxn02141 rxn02142 rxn03656 rxn04663 rxn06904 rxn07595 rxn15999 rxn16000 rxn16085 rxn16086 rxn20159
7:                                                                                           rxn03018 rxn03694
8:                                                                                                            
9:                                                                                                            
[1] "Reactions already in model:  rxn02141,rxn02142,rxn04663,rxn06904,rxn07595,rxn15999,rxn16000,rxn16085,rxn16086"
[1] "Reactions not in gapseq reaction database:  rxn03656,rxn20159,rxn31706,rxn31716"
[1] "Added reactions:  rxn02747,rxn03018,rxn03694,rxn03800,rxn03801,rxn04650,rxn04653,rxn07908,rxn07909,rxn15998,rxn16082,rxn16084"
Error in sybilSBML::writeSBML(mod, filename = paste0(out.id, ".xml"),  : 
  met_charge mustn't be NA
Calls: write_gapseq_sbml -> <Anonymous>
Execution halted
```
And with the newest version (17th June): **1.1 54632b3**

```
(gapseq) user@server:~/Programas/gapseq$ ./gapseq adapt add 14DICHLORBENZDEG-PWY toy/myb71.RDS 
Warning message:
In matrix(c("model", "m", 1, "character", "GapSeq model to be extended (RDS or SBML)",  :
  data length [24] is not a sub-multiple or multiple of the number of rows [5]
Error in getopt(spec) : 
  redundant short names for flags (column 2 of spec matrix).
Execution halted
```
In the second server I updated sybill since first there was an error with that.  I attached the test for both version.

I appreciate any help you can provide.

Best regards.

[gapseq_test_75ebaea.txt](https://github.com/jotech/gapseq/files/6830108/gapseq_test_75ebaea.txt)
[gapseq_test_54632b3.txt](https://github.com/jotech/gapseq/files/6830109/gapseq_test_54632b3.txt)

---
Esteban

**Comment by Waschina** [comment]
Hi Esteban,

thanks for pointing out the issue. The issue you had with 1.1 75ebaea should already be fixed (commits f40bba328c734be4614fedb006201724c22b00fd) and the new issue now also be fixed with commit (6ae3a6a6a85cef0b9c85bc58f997a59858e0f6df).

Please let me now if issues remain.

Best wishes
Silvio

**Comment by estebano11** [comment]
Hi Silvio, 

That was very quickly, thanks! It works perfectly now. 

Thanks!

Best regards.

--- 
Esteban
